db/config.cc

```diff
@@ -783,7 +783,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , role_manager(this, "role_manager", value_status::Used, "org.apache.cassandra.auth.CassandraRoleManager",
         "The role-management backend, used to maintain grantts and memberships between roles.\n"
         "The available role-managers are:\n"
-        "\tCassandraRoleManager : Stores role data in the system_auth keyspace.")
+        "\tCassandraRoleManager : Stores role data in the system_auth keyspace.\n"
+        "\tMaintenanceModeRoleManager : Allows all access to all users. This role-manager is used in the maintenance mode.")
```

Comment:
Why expose this? Maybe we should even make sure users cannot "accidentally" select this via the `role_manager` config.